src/components/apps/launch/index.js

```diff
@@ -71,9 +72,17 @@ export default ({ app, launchError, loading }) => {
     const defaultMaxMemory = config?.tools?.private.max_memory_limit;
     const defaultMaxDiskSpace = config?.tools?.private.max_disk_limit;
 
+    const baseId = "apps";
+
+    if (launchError) {
+        return (
+            <WrappedErrorHandler errorObject={launchError} baseId={baseId} />
+        );
+    }
```

Comment:
If we're returning this handler now for a `launchError`, then I guess it could be removed from the `appError` prop in the `AppLaunchWizard` below (it was being displayed in an `ErrorTypography` + `DEErrorDialog` by the `AppInfo` component before).